### PR TITLE
K8SPG-77 add support of PG14

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: e2e-tests/demand-backup/run [hunk 1]
```diff
@@ -101,7 +101,7 @@ function main {
 	kubectl_bin delete perconapgcluster/${cluster}
 	sleep 30
 
-	gsutil rm -r gs://${BUCKET}/backrestrepo/${cluster}-backrest-shared-repo
+	gsutil -m rm -r gs://${BUCKET}/backrestrepo/${cluster}-backrest-shared-repo
```
Comment (cap1984) [inline e2e-tests/demand-backup/run:104]: -m option could lead to gsutil infinite waiting time.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| demand-backup | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| scheduled-backup | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/151/commits/3ad122381e8d1675a342bf420083871b770267da
image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-151-3ad12238-pgo-deployer`

**Review by cap1984 — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| demand-backup | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| scheduled-backup | failed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/151/commits/f85e4d1417018e40f42234581c897d396e590280
image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-deployer`

File: e2e-tests/demand-backup/run (hunk 1 above)
Comment (hors) [reply e2e-tests/demand-backup/run:104]: I have tested it and it works. And we save +- 5-7 mins

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| clone-cluster | passed |
| scheduled-backup | passed |
| tls-check | passed |
| upgrade | passed |
| data-migration-gcs | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/151/commits/f85e4d1417018e40f42234581c897d396e590280
image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-151-f85e4d14-pgo-deployer`

**Pull request merged**